web-console/src/lib/components/streaming/builder/NodeTypes/InputNode.tsx

```diff
@@ -66,7 +69,19 @@ const InputNode = ({ id, data }: NodeProps<{ connector: ConnectorDescr }>) => {
         sx={{ display: 'flex', alignItems: 'center', position: 'absolute', top: 4, right: 4 }}
         className='nodrag nopan'
       >
-        <IconButton size='small' aria-label='close' sx={{ color: 'text.secondary' }} onClick={() => onDelete()}>
+        <IconButton
+          size='small'
+          aria-label='close'
+          sx={{ color: 'text.secondary' }}
+          onClick={() =>
+            showDeleteDialog(
+              'Remove',
+              `${data.connector.name || 'unnamed'} input`,
+              onDelete,
+              'You can add it back later.'
```

Comment:
Can you?

Reply:
The input node here is just removed from the pipeline in the Builder view, so can be added back from the drawer